tsfc/coffee.py

```diff
@@ -195,7 +190,7 @@ def statement_evaluate(leaf, parameters):
         table[abs(table + 1.0) < epsilon] = -1.0
         table[abs(table - 0.5) < epsilon] = 0.5
         table[abs(table + 0.5) < epsilon] = -0.5
-        init = coffee.SparseArrayInit(table, parameters.precision, nz_bounds)
+        init = coffee.ArrayInit(table, parameters.precision)
```

Comment:
Well, [this](https://gist.github.com/miklos1/ad89ebe7bdec1b0763e36f609209f928) has more compact tables, but code actually looks worse. I think it is better to just leave it like above for now.